code/controllers/subsystem/economy.dm

```diff
@@ -1,3 +1,5 @@
+#define VIP_BUDGET_BASE rand(8888888, 11111111)
```

Comment:
what does this do? Is this the amount of money they start with?

Reply:
Yes it is. I didn't want to put rand value there with no context.

Reply:
I mean. Money in their budget.

Reply:
They start with 8 million to 11 million credits?

Reply:
It's not VIPs that start with those credits. VIPs start with 10,000 credits and paid for 2,000 credits.
The budget card starts with that credits, but there's no way to pull it out when I took a look at all possible methods.

Reply:
I don't think they should be able to purchasen8 bicycles roundstart

Reply:
Can't they just roundstart buy 8 bicycles??

Reply:
It looks like this is their "inheritance" (if you think of them like children who inherited family money), they aren't roundstart given it, but it'll slowly trickle into their account as the round progresses.

Reply:
> Can't they just roundstart buy 8 bicycles??

Like mystery3525 said, they are not possible to access this budget unless 'admeme' really does something